pkg/netutil/cidr.go

```diff
@@ -3,9 +3,37 @@ package netutil
 import (
 	"net"
 
+	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/utils"
 )
 
+var (
+	privateSubnets = []*net.IPNet{
+		MustParseCIDR("127.0.0.0/8"),    // IPv4 localhost
+		MustParseCIDR("10.0.0.0/8"),     // class A
+		MustParseCIDR("172.16.0.0/12"),  // class B
+		MustParseCIDR("192.168.0.0/16"), // class C
+		MustParseCIDR("::1/128"),        // IPv6 localhost
+		MustParseCIDR("fd00::/8"),       // IPv6 ULA
+	}
+	log = logging.LoggerForModule()
+)
+
+// IsCIDRBlockInPrivateSubnet parses cidrStr and checks if it falls under the RFC 1819 private IP range
+func IsCIDRBlockInPrivateSubnet(cidrStr string) bool {
+	_, cidr, err := net.ParseCIDR(cidrStr)
+	if err != nil {
+		log.Warnf("Parsing incorrect CIDR block string (%s) as CIDR %s", cidrStr, err)
```

Comment:
Low-level library functions should not log, this can be very spammy.

Reply:
Pulled the warning up to the caller of this function.